### PR TITLE
Doc: graphical isochrones and global places

### PR DESCRIPTION
**Pull request by stifoon**

### CODE REVIEW

**Comment by stifoon** [comment]
viewable here: http://stifoon.github.io/navitia

**Comment by ZoeBrunet** [comment]
Ok for me

**Comment by stifoon** [comment]
Can someone take a look at this PR please?

**Comment by stifoon** [comment]
Can someone take a look at this PR please? @pbougue @TeXitoi @antoine-de

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -487,8 +487,16 @@ Differents kind of objects can be returned (sorted as):
...
 
 | url | Result |
-|------------------------------------------------|-------------------------------------|
-| `/coverage/places`                             | List of geographical objects        |
+|------------------------------------------------|-------------------------------------------------|
+| `/coverage/{region_id}/places`                 | List of geographical objects within a coverage  |
+| `/places`                                      | *Beta*: List of geographical objects within Earth |
+
+<aside class="warning">
+    Until now, you have to specify the right coverage to get `/places`.<br>
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md:495]: :+1:

File: documentation/slate/source/includes/apis.md [hunk 2]
```diff
@@ -591,58 +707,66 @@ The other one, the most used, is to access the 'journey' api endpoint:
...
-<aside class="success">
-    Neither the 'from' nor the 'to' parameter of the journey are required,
-    but obviously one of them has to be provided.
-    <br>
-    If only one is defined an isochrone is computed with every possible
-    journeys from or to the point.
-</aside>
+####Requesting an isochrone
+
+If you want to retrive every possible journeys form a single point at a time, you can request as:
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: `retrieve` every possible `journey` `from` a single point at a time, you can request as `follow`:

File: documentation/slate/source/includes/apis.md [hunk 3]
```diff
@@ -591,58 +707,66 @@ The other one, the most used, is to access the 'journey' api endpoint:
...
     Code it yourself on JSFiddle
 </a>
 
-### Accesses
-
-| url | Result |
-|------------------------------------------|-------------------------------------|
-| `/journeys`                          | List of journeys from wherever land |
-| `/coverage/{region_id}/journeys` | List of journeys on a specific coverage |
+The [isochrones](#isochrones) service exposes another response structure, which is simplier, for the same data.
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: `simpler`

File: documentation/slate/source/includes/apis.md [hunk 4]
```diff
@@ -751,6 +875,56 @@ direction       | int                    | Angle (in degree) between the previou
...
+
+
+This service gives you a multi-polygon response which 
+represent a same time travel zone: https://en.wikipedia.org/wiki/Isochrone_map
+
+As you can find isochrone tables using `/journeys`, this service is only another representation 
+of the same data, map oriented.
+
+It is also really usefull to make filters on geocoded objects to find which of 
+them are reachable beneath a specific time. You just have to verify that coordinates
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: `within` specific time? not sure though

Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: that `the` coordinates

File: documentation/slate/source/includes/isochrones.md [hunk 5]
```diff
@@ -4,9 +4,18 @@
...
-the quickest way to reach your destination. 
-The Isochrone function provides information in the form of a table with all the possible stops 
-from a potential destination with their respective arrival times, travel times and number of matches.
+the quickest way to reach your destination.
+
+![image](isochrones_example.png)
+
+The Isochrone function provides information either:
+
+-   in the form of a list with all the reachable stops from a potential destination
```
Comment (pbougue) [inline documentation/slate/source/includes/isochrones.md]: maybe you can mention the corresponding apis for each?

File: documentation/slate/source/includes/isochrones.md [hunk 6]
```diff
@@ -15,9 +24,11 @@ from a potential destination with their respective arrival times, travel times a
     Code it yourself on JSFiddle
 </a>
 
-You have to use these APIs (click on them for details):
+You could use these APIs (click on them for details):
```
Comment (pbougue) [inline documentation/slate/source/includes/isochrones.md]: I'd say `can`

File: documentation/slate/source/index.md [hunk 7]
```diff
@@ -8,8 +8,8 @@ toc_footers:
   - <a href='http://navitia.io/register/'>Get an access</a>
   - <a href='http://www.navitia.io/datasets/'>Explore Data</a>
   - <a href='https://groups.google.com/forum/#!forum/navitia'>Help !</a>
-
   - <a href='http://navitia.io'>Navitia.io</a>
+  - <a href='http://canaltp.github.io/navitia-playground'>Navitia Playground</a>
```
Comment (pbougue) [inline documentation/slate/source/index.md:12]: :+1:

**Comment by stifoon** [comment]
poke @pbougue

File: documentation/slate/source/includes/apis.md [hunk 8]
```diff
@@ -751,6 +875,55 @@ direction       | int                    | Angle (in degree) between the previou
...
+</aside>
+
+
+This service gives you a multi-polygon response which 
+represent a same time travel zone: https://en.wikipedia.org/wiki/Isochrone_map
+
+As you can find isochrone tables using `/journeys`, this service is only another representation 
+of the same data, map oriented.
+
+It is also really usefull to make filters on geocoded objects in order to find which ones are reachable within specific times.
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: `a specific time.` i'd say...

**Pull request merged**